editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -39,15 +39,16 @@ class AcceptDialog;
 class CanvasItemEditorViewport;
 class ConfirmationDialog;
 class EditorData;
-class EditorZoomWidget;
 class EditorSelection;
+class EditorZoomWidget;
 class HScrollBar;
 class HSplitContainer;
 class MenuButton;
 class PanelContainer;
-class ViewPanner;
+class StyleBoxTexture;
 class VScrollBar;
 class VSplitContainer;
+class ViewPanner;
```

Comment:
Did you move it because capital letters should go before lower case letters? I'm not sure it's worth following that.

Reply:
I use https://github.com/Tyriar/vscode-sort-lines for sorting lines and apparently it does follow this weird rule, with no way for configuration :/
Nonetheless, fixed it.